.ci/jenkins/dsl/jobs.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+* This file is describing all the Jenkins jobs in the DSL format (see https://plugins.jenkins.io/job-dsl/)
+* needed by the Kogito pipelines.
+*
+* The main part of Jenkins job generation is defined into the https://github.com/kiegroup/kogito-pipelines repository.
+*
+* This file is making use of shared libraries defined in
+* https://github.com/kiegroup/kogito-pipelines/tree/main/dsl/seed/src/main/groovy/org/kie/jenkins/jobdsl.
+*/
+
+import org.kie.jenkins.jobdsl.model.JenkinsFolder
+import org.kie.jenkins.jobdsl.model.JobType
+import org.kie.jenkins.jobdsl.utils.EnvUtils
+import org.kie.jenkins.jobdsl.utils.JobParamsUtils
+import org.kie.jenkins.jobdsl.KogitoJobTemplate
+import org.kie.jenkins.jobdsl.KogitoJobUtils
+import org.kie.jenkins.jobdsl.Utils
+
+jenkins_path = '.ci/jenkins'
+
+///////////////////////////////////////////////////////////////////////////////////////////
+// kie-benchmarks jobs
+///////////////////////////////////////////////////////////////////////////////////////////
+
+// update drools/optaplanner version in kie-benchmarks
+setupDroolsUpdateJob()
+setupOptaPlannerUpdateJob()
```

Comment:
```suggestion
setupUpdateDependencyJob('optaplanner')
setupUpdateDependencyJob('drools')
```

Those 2 methods are linked to one Jenkinsfile. You could make it more generic (goes with the `DEPENDENCY_PROJECT` comment)
and have a single method `setupUpdateDependencyJob`